bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableTable.java

```diff
@@ -161,6 +161,7 @@ public void batch(List<? extends Row> actions, Object[] results)
     batchExecutor.batch(actions, results);
   }
 
+  @Deprecated
```

Comment:
I'll add an {@inheritDoc} which should load in the docs from Table which explain why this method is deprecated.